src/columnresizing.js

```diff
@@ -287,8 +290,59 @@ function handleDecorations(state, cell) {
       const pos = start + cellPos + table.nodeAt(cellPos).nodeSize - 1;
       const dom = document.createElement('div');
       dom.className = 'column-resize-handle';
+      dom.dataset.test = 'test-column-resize-handle'
       decorations.push(Decoration.widget(pos, dom));
     }
   }
   return DecorationSet.create(state.doc, decorations);
 }
+
+const CELL_PADDING = 10;
```

Comment:
That's should be configurable by the user of this package, the width of the cell padding and the sort button width are define in the editor code.

For now let's just add "TODO" comment that ask to add it to the params the package users pass like `cellMinWidth `